content/en/docs/reference/_index.md

```diff
@@ -19,7 +21,19 @@ This section of the Kubernetes documentation contains references.
 ## API Reference
 
 * [Kubernetes API Reference {{< latest-version >}}](/docs/reference/generated/kubernetes-api/{{< latest-version >}}/)
-* [Using The Kubernetes API](/docs/reference/using-api/) - overview of the API for Kubernetes.
+* [Kubernetes API Overview](/docs/reference/using-api/) - overview of the API for Kubernetes.
```

Comment:
Could do this:
```suggestion
## Kubernetes API
* [API overview](/docs/reference/using-api/)
* [Kubernetes API reference {{< latest-version >}}](/docs/reference/generated/kubernetes-api/{{< latest-version >}}/)
* [API access control](/docs/reference/access-authn-authz/) - details on how Kubernetes controls API access
* [Well-Known Labels, Annotations and Taints](/docs/reference/kubernetes-api/labels-annotations-taints/)
```

(possibly omitting the heading line)
If you make this change @raghvenders, remove the “API Access Control” section.

Reply:
@sftim  - I like this by doing this is it fine to make the left navigator link go away too for API Reference(API access control & Well-Known Labels, Annotations, and Taints) and remove API Access control from the left navigation section?